README.md

```diff
@@ -13,6 +13,7 @@ Options:
      By default 10; can't be negative.
 -c   Max number of requests to be made at any time.
      By default 10; can't be negative or zero.
+-r   Report latency for an individual region.
```

Comment:
Maybe we can allow -r to be a list of regions.

Reply:
Maybe, we shouldn't. It becomes easier to parse the output if it only reports one region.

Reply:
@rakyll I prefer one region report too. What we could do is either to create a new flag `regions` to list supported regions, and let other scripts to consume & filter the regions. Alternatively, add a new flag `filter` where we can run gcping based on a regex `"us-*"`, `"*-east* "`. WDYT?

Reply:
Agreed.